tests/unit/functions/getIvaDefectsByManual.unitTest.ts

```diff
@@ -0,0 +1,73 @@
+const mockValidateIvaDefectManualQuery = jest.fn();
```

Comment:
I think these tests are good and needed, but I think it would be more beneficial and help us to validate if they were done in the same way the existing 'functional' tests were done. Using the .intTest and actually spinning up the local instance etc.

Reply:
They are unit tests that test the handler functions, not functional a.k.a integration tests.

Reply:
Fair enough, is it possible to also add the .int Tests as well then please?

Reply:
I've added [CB2-9900 ](https://dvsa.atlassian.net/browse/CB2-9900) to add in int tests for this. Would like to merge this to the holding branch before those tests so as to progress the infrastructure work.